runner/generate_helper_sql.go

```diff
@@ -13,7 +13,8 @@ import (
 
 var statsHelpers = []string{
 	// Column stats
-	`CREATE OR REPLACE FUNCTION pganalyze.get_column_stats() RETURNS TABLE(
+	`DROP FUNCTION IF EXISTS pganalyze.get_column_stats;
+CREATE OR REPLACE FUNCTION pganalyze.get_column_stats() RETURNS TABLE(
```

Comment:
```suggestion
CREATE FUNCTION pganalyze.get_column_stats() RETURNS TABLE(
```
NBD, but we don't need the "OR REPLACE" if we DROP IF EXISTS right beforehand.